docs/keto/guides/rbac.mdx

```diff
@@ -2,6 +2,234 @@
 title: Role Based Access Control (RBAC)
 ---
 
-This guide will explain how to implement RBAC using Ory Keto. It's still work in
-progress. If you happened to figure this out yourself, please open a PR to add
-your findings on this page.
+This guide will explain how to implement RBAC using Ory Keto.
+
+:::warning
+
+Implementing RBAC currently is possible, but requires some workarounds. This
+guide enables RBAC support with Keto but native support is still work in
+progress. Follow the progress in
+[this issue](https://github.com/ory/keto/issues/598)
+:::
+
+[Role Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control)
+maps subjects to roles and roles to permissions. The goal of (H)RBAC is to make
+permission management convenient by grouping subjects in roles and assigning
+permissions roles. This type of access control is common in web applications
+where one often encounters roles such as "administrator", "moderator", and so
+on.
+
+In **Hierarchical Role Based Access Control (HRBAC)** roles can inherit
+permissions from other roles. The "administrator" role, for example, could
+inherit all permissions from the "moderator" role. This reduces duplication and
+management complexity around defining privileges.
+
+Let's assume that we are building a reporting application and need to have three
+groups of users with different access levels.
+We have the following group of reports in our application.
+
+- Financial performance reports
+- Marketing performance reports
+- Community performance reports
+
+This time we model the access rights using (H)RBAC and the roles
+`community`, `marketing`, `finance` and `admin`:
+
+The role `admin` inherits all privileges from `finance`, `marketing` and `community`
+
+(H)RBAC is everywhere. If you ever installed a forum software such as
+[phpBB](https://www.phpbb.com/support/docs/en/3.1/ug/adminguide/permissions_roles/)
+or [Wordpress](https://codex.wordpress.org/Roles_and_Capabilities), you have
+definitely encountered ACL, (H)RBAC, or both.
+
+(H)RBAC reduces management complexity and overhead with large user bases.
+Sometimes however, even (H)RBAC is not enough. An example is when you need to
+express ownership (e.g. `Dilan` can only modify his own reports), have
+attributes (e.g. `Dilan` needs to have access only during work hours),
+or in multi-tenant environments.
+
+**Benefits:**
+
+- Reduces management complexity when many identities share similar permissions.
+- Role hierarchies can reduce redundancy even further.
+- Is well established and easily understood by many developers as it is a
+  de-facto standard for web applications.
+
+**Shortcomings:**
+
+- Has no concept of context:
```

Comment:
It looks like the second half is missing here. I think we can just delete this line as you already explain "context" in the next bullet points, right?

Reply:
right